src/java.desktop/share/classes/javax/swing/border/TitledBorder.java

```diff
@@ -34,7 +34,6 @@
 import java.awt.Rectangle;
 import java.awt.geom.Path2D;
 import java.beans.ConstructorProperties;
-import java.beans.PropertyChangeEvent;
 import java.beans.PropertyChangeListener;
```

Comment:
Is removing an unused import the only change to TitledBorder ? I think you should revert this to make the change cleaner.